object-details/design.md

```diff
@@ -0,0 +1,29 @@
+# Object details
+
+
+## Sidecar field
+![](img/sidecar-suggestion.png)
+- We could add a `sidecar` field to the item details to illustrate whether the sidecar is missing or represent
```

Comment:
I think this it the less optimal place to be part of, and at the bottom of the details. 
It is also conceptually not "the same" as the other details here, because it is something "applied to" and not really "part of" the current entity.

Reply:
I am in favor of either putting the sidecar indicator in line with the header or below the header, but not as part of details section.